cmd/root.go

```diff
@@ -79,6 +79,8 @@ any client SSL certificates.`,
 	// Global-only flags
 	cmd.PersistentFlags().StringVarP(&c.conf.Token, "token", "t", "",
 		"Bearer token used for authorization.")
+	cmd.PersistentFlags().StringVarP(&c.conf.CredentialsFile, "credentials-file", "c", "",
```

Comment:
does "c" make sense here, or should it be "cf"?

Reply:
cf sounds good to me.

Reply:
I like the single letter if only to match what I imagine most CLIs do -- long phrase for a flag and a corresponding single letter. So far, I've been just using whatever the long phrases starts with to pick a single letter.